contributor/making-release.md

```diff
@@ -16,12 +52,15 @@ There are number of things that are required before attempting to build a releas
 2. Ensure that you're using the correct major release of Java (check javadoc too).
 3. Ensure that you're building Apache Accumulo with a username that has the same name as your Apache ID (this is due to
    the maven-release-plugin and staging the release candidate).
-4. Update the CHANGES file so that it's in sync with Jira (manual process).
-5. Ensure that you have a texlive distribution installed so you are able to build the documentation.
-6. Have a clean workspace before starting.
+4. Ensure that you have a texlive distribution installed so you are able to build the documentation.
```

Comment:
We don't use LaTeX any more. We use asciidoc for 1.9, and 2.0 has docs directly on the website.